tests/test_supply_curve_tech_mapping.py

```diff
@@ -23,13 +28,78 @@
 TM_DSET = 'techmap_nsrdb_ri_truth'
 
 
-def test_resource_tech_mapping():
+@pytest.mark.parametrize("batch_size", [100, 50])
+def test_resource_tech_mapping(tmp_path, batch_size):
     """Run the supply curve technology mapping and compare to baseline file"""
-    ind = TechMapping.run(EXCL, RES, dset=None, max_workers=2)
+
+    excl_fpath = EXCL
+    excl_fpath = tmp_path.joinpath("excl.h5").as_posix()
+    shutil.copy(EXCL, excl_fpath)
+
+    dset = "tm"
+    TechMapping.run(
+        excl_fpath, RES, dset=dset, max_workers=2, sc_resolution=2560,
+        batch_size=batch_size
+    )
 
     with ExclusionLayers(EXCL) as ex:
         ind_truth = ex[TM_DSET]
 
+    with ExclusionLayers(excl_fpath) as out:
+        assert dset in out, "Techmap dataset was not written to H5"
+        ind = out[dset]
+
+    msg = 'Tech mapping failed for {} vs. baseline results.'
+    assert np.allclose(ind, ind_truth), msg.format('index mappings')
```

Comment:
Am I crazy or can we skip the message formatting here?

```suggestion
    msg = 'Tech mapping failed for index mappings vs. baseline results.'
    assert np.allclose(ind, ind_truth), msg
```